tests/timezones/models.py

```diff
@@ -25,3 +25,7 @@ class Timestamp(models.Model):
 
 class AllDayEvent(models.Model):
     day = models.DateField()
+
+
+class ArbitraryTimestamp(models.Model):
```

Comment:
This model has the same fields as `MaybeEvent` (I'm not sure that the minimal test requires a nullable field though).